ads/yieldmo.md

```diff
@@ -0,0 +1,36 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Yieldmo
+
+## Example
+
+```html
+<amp-ad width=300 height=168
+    type="yieldmo"
+    data-ymid="1349317029731662884"
+    data-ymv="2">
```

Comment:
This is actually our internal swimlane version. Our internal admin tool assigns the publisher a version and gives them that code snippet to paste into their websites.